types/redux-saga-tester/index.d.ts

```diff
@@ -21,6 +23,11 @@ export interface SagaTesterOptions<StateType> {
 export default class SagaTester<StateType extends object> {
     constructor(options?: SagaTesterOptions<StateType>);
 
+    /**
+     * The reference of the redux store.
+     */
+    store: Store<StateType>;
```

Comment:
I don't see the store in the api, not sure it's a best practice to add typings for undocumented features?  https://www.npmjs.com/package/redux-saga-tester#api

Reply:
You are right that the store is not documented.

For testing purpose I use the `sagaTester.store` to hand it into the redux `Provider` like:

```js
<Provider store={sagaTester.store}>
    <SomeComponentWhichAccessTheReduxStore/>
</Provider>
```

which allows to use the sagaTester store as component store.

Reply:
do you mind making a PR into redux-saga-tester first to add it to the README?

Perhaps `store` is meant to be private, I don't know

Reply:
Since the redux-saga-tester package is abandoned (last commit on 25.06.2019 and no responses to open issues and PRs for some time) it won't be easy to improve the documentation and others things on that side.